frontend/packages/operator-lifecycle-manager/src/const.ts

```diff
@@ -8,3 +8,5 @@ export const OPERATOR_TYPE_ANNOTATION = 'operators.operatorframework.io/operator
 export const NON_STANDALONE_ANNOTATION_VALUE = 'non-standalone';
 export const INTERNAL_OBJECTS_ANNOTATION = 'operators.operatorframework.io/internal-objects';
 export const DEFAULT_SOURCE_NAMESPACE = 'openshift-marketplace';
+export const OPERATOR_PLUGINS_ANNOTATION = 'console.openshift.io/plugins';
+export const CONSOLE_OPERATOR_CONFIG_NAME = 'cluster';
```

Comment:
We might move this elsewhere since it's not specific to OLM. Maybe

https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/constants/resource.ts